### PR TITLE
Update advanced-features.md

### LINKED ISSUE #12393: add link request to "Endpoint Attack Notifications" page
Regarding the section of "Endpoint Attack Notifications".
https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/advanced-features?view=o365-worldwide#endpoint-attack-notifications

Please consider to put a link to the page explaining about "Endpoint Attack Notifications".
https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/endpoint-attack-notifications?view=o365-worldwide

---

```[tasklist]
### Tasks
- [ ] Add a draft title or issue reference here
```

#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: b3c074f5-bd58-6b72-d891-3d9fe0118767
* Version Independent ID: 2eea3a11-be82-49bc-6056-f4fba30bd638
* Content: [Configure advanced features in Microsoft Defender for Endpoint](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/advanced-features?view=o365-worldwide#endpoint-attack-notifications)
* Content Source: [microsoft-365/security/defender-endpoint/advanced-features.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/advanced-features.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @mjcaparas
* Microsoft Alias: **macapara**

### PR DESCRIPTION
**Pull request by msbemba**
Added Endpoint attack notifications URL to the document

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12393

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/advanced-features.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ To receive contextual device integration in Office 365 Threat Intelligence, you'
 
 ## Endpoint Attack Notifications
 
-Endpoint Attack Notifications enable Microsoft to actively hunt for critical threats to be prioritized based on urgency and impact over your endpoint data. 
+[Endpoint Attack Notifications](/security/defender-endpoint/endpoint-attack-notifications) enable Microsoft to actively hunt for critical threats to be prioritized based on urgency and impact over your endpoint data. 
 
 For proactive hunting across the full scope of Microsoft 365 Defender—including threats that span email, collaboration, identity, cloud applications, as and endpoints—[learn more](https://aka.ms/DefenderExpertsForHuntingGetStarted) about Microsoft Defender Experts.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/advanced-features.md]:
```suggestion
For proactive hunting across the full scope of Microsoft 365 Defender—including threats that span email, collaboration, identity, cloud applications, and endpoints—[learn more](https://aka.ms/DefenderExpertsForHuntingGetStarted) about Microsoft Defender Experts.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, please commit the additional change to correct the typo. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**